packages/react/src/components/progress/progress.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { Progress, ProgressProps } from '@onfido/castor-react';
+import React, { useEffect, useState } from 'react';
+import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
+
+const size = ['regular', 'large'] as const;
+
+export default {
+  title: 'React/Progress',
+  component: Progress,
+  argTypes: {
+    children: {
+      description: 'Optional label.',
+    },
+    value: {
+      description: 'Current value',
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    max: {
+      description: 'Optional maximum value',
+      table: {
+        defaultValue: { summary: '100' },
+      },
+    },
+    size: {
+      control: { type: 'radio', options: size },
+      table: {
+        type: { summary: size.toString() },
+        defaultValue: { summary: 'text' },
+      },
+    },
+    hideLabel: {
+      description: 'Hide label',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+  },
+  args: {
+    children: '',
+    hideLabel: false,
+    max: 100,
+    size: 'regular',
+    value: 25,
+  },
+  parameters: { display: 'flex' },
+} as Meta<ProgressProps>;
+
+export const Playground: Story<ProgressProps> = (props: ProgressProps) => (
+  <Progress {...props} />
+);
+
+export const Size = reactMatrix(Progress, { size });
+Size.argTypes = omit<ProgressProps>('size');
+
+export const CustomLabel = (props: ProgressProps) => <Progress {...props} />;
+CustomLabel.argTypes = omit<ProgressProps>('children');
+CustomLabel.args = {
+  children: 'Progress: 25%',
+};
+
+export const WithoutLabel = (props: ProgressProps) => <Progress {...props} />;
+WithoutLabel.argTypes = omit<ProgressProps>('children');
+WithoutLabel.args = {
+  children: null,
+  hideLabel: true,
+};
+
+export const Example = (props: ProgressProps) => {
```

Comment:
Can we figure out a better name? Arguably every story is an `Example` 🙈 
Perhaps `Progression` or `Value` or `ValueLoop`?

I also noticed the code shown does not update, because it's internal state.
I wonder if there's a way to update the story's props and in turn the shown code.
Perhaps there's a `Storybook controls` hook?

Reply:
I was thinking if I shouldn't remove this altogether, we don't have any "running" example for other components.

It's nice to see it working, but the "show code" only make sense if we can see the context around (like the useEffect hook, etc).

Trying to find a way to do that, but maybe I will remove it, what do you think?

Reply:
We mostly do static examples because of core/vanilla.

I like the idea of a dynamic example but would like the code to update alongside state.
For which we need to use props, I think.
Decorators might do it too.

I don't know, maybe it's not worth the effort right now, but it's definitely nice to have.

Reply:
I have added the example manually. Can you take a look? It isn't perfect as we have to change both when updating the test, but at least, when clicking on `copy` the user will get a "working" example, missing the imports only.

Let me know if this is OK, if isn't, I think I will remove it for now.
If you think is best to remove, I
In case you think this is a problem, I think

Reply:
On Storybook it looks good, before AllCombinations is better though, that should always be the last story.

On code, it looks... well, repetitive. Does converting the function to a string work?

```ts
docs: {
  code: ValueLoop.toString()
}
```

Reply:
I agree, converting the function to string gets the JS instead of the JSX code:

```
function ValueLoop(props) {
  var _useState = Object(react__WEBPACK_IMPORTED_MODULE_16__["useState"])(0),
      _useState2 = _slicedToArray(_useState, 2),
      value = _useState2[0],
      setValue = _useState2[1];

  Object(react__WEBPACK_IMPORTED_MODULE_16__["useEffect"])(function () {
    setTimeout(setValue, 1000, value === 100 ? 0 : value + 10);
  }, [value]);
  return /*#__PURE__*/Object(react_jsx_runtime__WEBPACK_IMPORTED_MODULE_18__["jsx"])("div", {
    "aria-busy": value !== 100,
    style: {
      width: '100%'
    },
    children: /*#__PURE__*/Object(react_jsx_runtime__WEBPACK_IMPORTED_MODULE_18__["jsx"])(_onfido_castor_react__WEBPACK_IMPORTED_MODULE_15__[/* Progress */ "a"], Object.assign({}, props, {
      value: value
    }))
  });
}
```

Maybe remove the example for now? I can try to find better solutions some other time.

Reply:
Ah that's too bad. I don't know, I would personally keep it I think.

@josokinas ?

Reply:
I like that this way a JS is shown.

However, for such example I don't believe that JS code is needed - it actually just a mockup to make a movement.

I like thought that `aria-busy` is used, so if the example by default renders a JSX and shows both aria and value changed, I would be happy with just it. Otherwise we can live with code duplication.

Reply:
Not sure I got it. So keep it?

Reply:
Keep it.